projects/quadrat/index.js

```diff
@@ -0,0 +1,61 @@
+const { GraphQLClient, gql } = require('graphql-request')
+const sdk = require('@defillama/sdk')
+
+const CHAINS = ['polygon', 'optimism', 'ethereum', 'arbitrum'];
+
+const GRAPH_CLIENTS = {
+  polygon: new GraphQLClient('https://api.thegraph.com/subgraphs/name/krivochenko/plasma-quadrat-matic'),
+  optimism: new GraphQLClient('https://api.thegraph.com/subgraphs/name/krivochenko/plasma-quadrat-optimism'),
+  ethereum: new GraphQLClient('https://api.thegraph.com/subgraphs/name/krivochenko/plasma-quadrat-mainnet'),
+  arbitrum: new GraphQLClient('https://api.thegraph.com/subgraphs/name/krivochenko/plasma-quadrat-arbitrum'),
+};
+
+const GRAPH_QUERY = gql`
+    query strategies ($timestamp: Int! $skip: Int! $first: Int!) {
+        strategies (skip:$skip first:$first) {
+            token0 {
+                id
+            }
+            token1 {
+                id
+            }
+            daysData (orderBy: createdAt orderDirection: desc skip:0 first: 2,where: {createdAt_lte:$timestamp}) {
```

Comment:
Hi, the PR is mostly good except this part, if the graph stops updating/indexing new blocks, we wll be showing stale tvl right? can you add a `createdAt_gte` as well, like two days before current timestamp.

Reply:
@ilyamk this issue still exists right?

Reply:
@g1nt0ki it should be fixed in PR #3